users/views.py

```diff
@@ -66,3 +67,24 @@ def post(self, request):
         except User.DoesNotExist:
             return JsonResponse({'message' : 'INVALID_USER'}, status=401)
 
+class MemberShipOutView(View):
+    @signin_decorator
+    def delete(self, request):
+        try:
+            data     = json.loads(request.body)
+            user     = request.user
+            password = data['password']
+            email    = data['email']
+
+            if not User.objects.filter(id = user.id, email = email).exists():
+                return JsonResponse({'message' : 'USER_NOT_EXISTS'}, status=404)
```

Comment:
email과 비밀번호 중 어떤 것이 잘못 되었는지 모르게 해주세요.
```suggestion
            if not User.objects.filter(id = user.id, email = email).exists():
                return JsonResponse({'message' : 'INVALID_USER'}, status=404)
```